### ISSUE TITLE
Bad quality?

### ISSUE THREAD

**Issue by SohaibJKhan** [carykh/jumpcutter#104]
When I run the program and set the quality to 1, I don't get source quality. Maybe it's throttling on lower spec systems? I'm using flv in OBS for recording.

**Comment by torsrex** [comment]
Not sure if you're referring to video or audio quality, but if you're referring to audio quality try this https://github.com/torsrex/jumpcutter/blob/master/jumpcutter.py . I added a parameter (--stretch_algorithm) to set the audio stretching algorithm. In most cases I got better quality audio by using "wsola" (I set this to be the default) instead of "phasevocoder".

**Comment by SohaibJKhan** [comment]
No, I was talking about video quality. Not sure if it's only on my end though. Appreciate the reply though, it might be useful down the line.

**Comment by kcomain** [comment]
Hey there, if you have fixed your problem, consider closing it. :)

**Comment by SohaibJKhan** [comment]
Nah, it's still an issue

**Comment by yunjiewang** [comment]
[Try setting qmin to 1 and see if it helps.](https://stackoverflow.com/questions/10225403/how-can-i-extract-a-good-quality-jpeg-image-from-an-h264-video-file-with-ffmpeg)
[Otherwise, use lossless format.](https://superuser.com/questions/1032503/ffmpeg-extract-lossless-image-from-video)

**Comment by cognitiveosman** [comment]
The issue is with the video encoder used in the tool, replace line 200 in the script with: 
`command = "ffmpeg -framerate "+str(frameRate)  +" -i "+TEMP_FOLDER+"/newFrame%06d.jpg -i "+TEMP_FOLDER+"/audioNew.wav -strict -2 " + " -c:v libx264 -pix_fmt yuv420p -crf 0 -c:a aac -b:a 128k "+ OUTPUT_FILE`
